accounts-db/src/read_only_accounts_cache.rs

```diff
@@ -102,21 +102,23 @@ impl ReadOnlyAccountsCache {
     pub(crate) fn new(
         max_data_size_lo: usize,
         max_data_size_hi: usize,
-        ms_to_skip_lru_update: u32,
+        evict_sample_size: usize,
     ) -> Self {
         assert!(max_data_size_lo <= max_data_size_hi);
+        assert!(evict_sample_size > 0);
         let cache = Arc::new(DashMap::with_hasher(AHashRandomState::default()));
-        let queue = Arc::new(Mutex::<IndexList<ReadOnlyCacheKey>>::default());
         let data_size = Arc::new(AtomicUsize::default());
         let stats = Arc::new(AtomicReadOnlyCacheStats::default());
+        let timer = Instant::now();
         let evictor_exit_flag = Arc::new(AtomicBool::new(false));
+        #[cfg(not(test))]
```

Comment:
I'm confused by this change, do we have no tests that check bg eviction?

Reply:
We do, good point. I removed that and I'm keeping the `#[cfg(not(test))]` just in `spawn_evictor`.